src/webui/webapplication.h

```diff
@@ -43,12 +43,11 @@
 #include "base/utils/net.h"
 #include "base/utils/version.h"
 
-constexpr Utils::Version<int, 3, 2> API_VERSION {2, 8, 2};
-
 class APIController;
 class WebApplication;
 
-constexpr char C_SID[] = "SID"; // name of session id cookie
+inline const Utils::Version<int, 3, 2> API_VERSION {2, 8, 3};
+inline const char C_SID[] = "SID"; // name of session id cookie
```

Comment:
I wonder if it is used anywhere other than `webapplication.cpp`? If not, why not move this declaration there?